app/models/gameworld_builder.rb

```diff
@@ -47,7 +51,9 @@ def create_spawns
       ((i + 1) % distance_between_spawns).zero?
     end
 
-    all_spawns.pop if all_spawns.size > @player_amount
+    while all_spawns.size > @player_amount
+      all_spawns.pop 
+    end
 
     all_spawns.sort_by { |s| s.y && s.x }.each do |p|
```

Comment:
```suggestion
    all_spawns.sample(@player_amount).sort_by { |s| s.y && s.x }.each do |p|
```
nit: could even use `sample` here

Reply:
very nice, habs angepasst